beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockFactoryDenebTest.java

```diff
@@ -42,6 +46,9 @@ public class BlockFactoryDenebTest extends AbstractBlockFactoryTest {
   private final Spec spec = TestSpecFactory.createMinimalDeneb();
   private final DataStructureUtil dataStructureUtil = new DataStructureUtil(spec);
 
+  private final StubMetricsSystem metricsSystem = new StubMetricsSystem();
```

Comment:
You could add both to `AbstractBlockFactoryTest` and avoid duplicating it in `BlockFactoryDenebTest`, `BlockFactoryFuluTest`, `BlockFactoryPhase0Test`. Check `attestationsPool`definition for example